src/components/Imports/Providers/nextcloud/accountService.js

```diff
@@ -0,0 +1,31 @@
+export async function createNextcloudAccount(client, { login, password, url }) {
+  if (!login || !password || !url) {
+    throw new Error('Missing credentials for Nextcloud account creation')
+  }
+
+  const accountData = {
+    _type: 'io.cozy.accounts',
+    account_type: 'nextcloud',
+    auth: {
+      login,
+      password,
+      url
+    },
+    identifier: 'login',
+    name: login,
+    state: null
+  }
+
+  const res = await client.save(accountData)
+  // cozy-client usually returns the saved doc directly, but keep a safe fallback
```

Comment:
We should be confident with cozy-client. No need to have "safe fallback". If it worked => you'll have res.data, if not you'll have an exception